editor/pom.xml

```diff
@@ -67,5 +67,29 @@
       <classifier>tests</classifier>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.hamcrest</groupId>
+      <artifactId>hamcrest-all</artifactId>
+      <version>1.3</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>net.bytebuddy</groupId>
+      <artifactId>byte-buddy-agent</artifactId>
+      <version>1.12.16</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>net.bytebuddy</groupId>
```

Comment:
If this built and ran fine without the byte-buddy and objenesis dependencies, let's leave them out.  Maven should get them on its own, but sometimes doesn't.

Reply:
yeah byte-buddy is required otherwise we get java.lang.IllegalStateException and removal of objenesis passes